src/esp/physics/RigidScene.h

```diff
@@ -19,31 +19,58 @@ class RigidScene : public RigidBase {
   /**
    * @brief Virtual destructor for a @ref RigidScene.
    */
-  virtual ~RigidScene(){};
+  virtual ~RigidScene() {}
 
   /**
-   * @brief Initializes the @ref RigidObject or @ref RigidScene that inherits
+   * @brief Initializes the @ref RigidScene that inherits
    * from this class
-   * @param physicsAttributes The template structure defining relevant
+   * @param resMgr a reference to ResourceManager object
+   * @param handle The handle for the template structure defining relevant
    * phyiscal parameters for this object
    * @return true if initialized successfully, false otherwise.
    */
-  virtual bool initialize(
-      const assets::ResourceManager& resMgr,
-      const assets::AbstractPhysicsAttributes::ptr physicsAttributes) override;
+  bool initialize(const assets::ResourceManager& resMgr,
+                  const std::string& handle) override;
+
+  /**
+   * @brief Get the template used to initialize this object or scene.
+   *
+   * AbstractPhysicsAttributes templates are able to be changed between
+   * instances of objects.
+   * @return The initialization settings of this object instance, casted to the
+   * appropriate value.
```

Comment:
Just an opinion, feel free to disregard.
```suggestion
   * @brief Get the template used to initialize this scene.
   *
   * See @ref RigidBase::getInitializationAttributes.
   * @return The initialization settings of this scene instance.
```